app/models/game.rb

```diff
@@ -1,5 +1,5 @@
 class Game < ApplicationRecord
-  # after_create :populate_game
```

Comment:
On this you may actually want this commented out, because if it's not, then all the tests will have a fully populated game board, which will make things more difficult to test.